### PR TITLE
Devel/buster fixup

### PR DESCRIPTION
**Pull request by iluminat23**
This is the devel/buster branch rebased to the current master.
+ reverted b161191 (add the libgcrypt20-dev)

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -2,11 +2,10 @@ Create custom images for Revolution Pi
...
 Intended usage:
 
-* Download Raspbian image, works with both desktop and lite images:
-  curl -O http://downloads.raspberrypi.org/raspbian/images/raspbian-2019-04-09/2019-04-08-raspbian-stretch.zip
-  curl -O http://downloads.raspberrypi.org/raspbian_lite/images/raspbian_lite-2019-04-09/2019-04-08-raspbian-stretch-lite.zip
-  unzip *-raspbian-*.zip
-  rm *-raspbian-*.zip
+* Download RaspiOS (previously called Raspbian) image, works with both desktop and lite images:
+  curl -O http://downloads.raspberrypi.org/raspios_armhf/images/raspios_armhf-2020-08-24/2020-08-20-raspios-buster-armhf.zip
+  curl -O http://downloads.raspberrypi.org/raspios_lite_armhf/images/raspios_lite_armhf-2020-08-24/2020-08-20-raspios-buster-armhf-lite.zip
```
Comment (zhan-kunbus) [inline README:7]: now it should be the raspios of Jan 2021

Comment (iluminat23) [reply README:7]: We can do this on the master

**Comment by zhan-kunbus** [comment]
looks good to me

**Review by zhan-kunbus — APPROVED** [review]
lgtm

**Pull request merged**